src/main/java/org/radarcns/hdfs/util/commandline/CommandLineArgs.java

```diff
@@ -93,6 +93,9 @@
     @Parameter(names = {"--max-files-per-topic"}, description = "Maximum number of records to process, per topic. Set below 1 to disable this option.")
     public int maxFilesPerTopic = 0;
 
+    @Parameter(names = {"--exclude-topics"}, description = "Topics to exclude when processing the records.")
```

Comment:
Parameter name should be singular, since on the command line you specify one value per tag, e.g. `--exclude-topic topicA --exclude-topic topicB`.

Reply:
It works as a comma-separated list. Is is preferred to use flag for each topic ?

Reply:
Based on JCommander docs, that is what I thought. I also think it is more clear from CLI point of view.

Reply:
Okay I will update.